### PR TITLE
ci: add gh action workflow definition for F5 CLA automation

### PR DESCRIPTION
**Pull request by TasoOneAsia**
### Proposed changes

Adds a GH action workflow file (`.github/workflows/f5-cla.yml`) for F5 contributor license agreement (CLA) automation and compliance. This action is configured to target a shared signature store for F5 Open Source projects and will prompt contributors, upon pull request to `main`, to sign the F5 CLA if they have not already done so. 

As part of the configuration for this project, the following GitHub usernames were whitelisted as maintainers:

* `Dean-Coakley`
* `pleshakov` 
* `lucacome`

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-prometheus-exporter/blob/main/CONTRIBUTING.md)
  guide
- [x] I have proven my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have ensured the README is up to date
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch on my own fork

### CODE REVIEW

File: .github/workflows/f5-cla.yml
Comment (lucacome) [reply .github/workflows/f5-cla.yml:35]: I don't think any other employees have contributed regularly in the past few years...but this doesn't seem very scalable. Is there a way to have a central configuration with a list of all the employees?

**Review by lucacome — APPROVED** [review]
I'm not a fan of CLAs, but 🤷‍♂️

File: .github/workflows/f5-cla.yml
Comment (TasoOneAsia) [reply .github/workflows/f5-cla.yml:35]: I agree. Ideally, we would have a master list, updated regularly, of GitHub usernames associated with active F5 employees to check against; however, such a list does not currently exist, and creating one has its own challenges.

Another option considered was inspecting the "email" field of the Git commit metadata and looking for the "f5.com" domain. However, this also assumes that employees are making all commits for F5 projects with their employee email (which, although outlined in the policy, isn't consistent enough practically to rely on)

**Comment by TasoOneAsia** [comment]
@lucacome Should this get any additional reviews, or is this good to go?

**Comment by lucacome** [comment]
@TasoOneAsia we require two approvals

**Comment by pdabelf5** [comment]
Should this PR update the contributing guide to advise how to get approval for the F5 CLA?

**Comment by TasoOneAsia** [comment]
Since this got the two approvals, I'll go ahead and squash + merge this on Monday if there are no objections.

**Comment by lucacome** [comment]
@TasoOneAsia what is the status of this?

File: .github/workflows/f5-cla.yml
Comment (lucacome) [inline .github/workflows/f5-cla.yml:34]: Not sure if former employees need to sign the CLA or not, but we should probably remove Dean.

**Comment by lucacome** [comment]
I'm going to close this since it's been open for 3+ months and there's no activity. Feel free to re-open it.

**Comment by TasoOneAsia** [comment]
@lucacome Sorry about the delaying in circling back to this, I'll update the branch to reflect a couple small changes that were made to the workflow recently and open up a new PR.

**Comment by lucacome** [comment]
sounds good @TasoOneAsia thanks!

**Pull request closed without merging**
